nise/generators/azure/azure_generator.py

```diff
@@ -381,22 +418,21 @@ def _update_data(self, row, start, end, **kwargs):
         row = self._map_header_to_report_version(
             row, meter_sub, str(amount), str(rate), str(cost), instance_id, service_tier
         )
-        if self.azure_columns == AZURE_COLUMNS_V2:
-            row = self.add_v2_specific_columns(row, instance_id=instance_id)
-        self._add_tag_data(row)
 
-        return row
-
-    def add_v2_specific_columns(self, row, **kwargs):
-        """Add values for columns specific to the V2 report."""
+        if self.azure_columns == AZURE_COLUMNS_V2_SUBSCRIPTION:
```

Comment:
This is comparing two potentially different tuples where order matters. If they contain the same values but are in a different order, the equality check will fail.

```
In [112]: (1,2) == (2,1)
Out[112]: False

In [113]: (1,2) == (1,2)
Out[113]: True
```

An identity check (`is`) might make more sense here, or at least sorting the values for more reliable comparison.

Reply:
Suggestion applied.